react/src/utils/Type.js

```diff
@@ -30,6 +30,10 @@ function IsNonEmptyArray(value) {
     return Array.isArray(value) && value.length > 0;
 }
 
+function IsJson(value) {
+    return IsObject(value) || IsArray(value);
```

Comment:
May not be practical but does `IsArray` really guarantee this is JSON? Couldn't it be some other inner value that is not JSON?

Reply:
Not sure what you mean, about "some inner value" ...